webapp/src/shared/types.ts

```diff
@@ -28,3 +30,12 @@ export type Avatar = {
   avatarX: number;
   avatarY: number;
 };
+
+type PlayState = 'playing' | 'paused';
+
+export type GameInfo = {
```

Comment:
Actually this one is missing some of the fields that that one has.
What about creating a common one, and have an intersection type with the common one + the extra ones in game.service.ts, and just the common one here?

Reply:
Updated. I've moved the types to the pong engine and split them into `GameInfoClient` and `GameInfoServer`. Naming suggestions are appreciated :smile: